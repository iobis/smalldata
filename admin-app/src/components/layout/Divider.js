import PropTypes from 'prop-types'
import React from 'react'

export default function Divider({ children }) {
  return (
    <div className="divider">
      <span>{children}</span>
    </div>
  )
}

Divider.propTypes = {
  children: PropTypes.node
}
