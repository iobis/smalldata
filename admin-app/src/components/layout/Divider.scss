.divider {
  width: 100%;
  text-align: center;
  border-bottom: 1px solid $grey-lighter;
  line-height: 0.1em;

  span {
    background: $white;
    padding: 0 10px;
  }
}
