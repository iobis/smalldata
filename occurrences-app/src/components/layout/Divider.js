import React from 'react'

export default function Divider({ children }) {
  return (
    <div className="divider">
      <span>{children}</span>
    </div>
  )
}
