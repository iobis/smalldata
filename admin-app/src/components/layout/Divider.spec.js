import Divider from './Divider'
import React from 'react'
import renderer from 'react-test-renderer'

describe('Divider', () => {
  it('renders correctly', () => {
    expect(renderer.create(<Divider>some text</Divider>).toJSON()).toMatchSnapshot()
  })
})
