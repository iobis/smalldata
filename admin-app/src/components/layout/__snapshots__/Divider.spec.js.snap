// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider renders correctly 1`] = `
<div
  className="divider"
>
  <span>
    some text
  </span>
</div>
`;
